tools/profile_model_ops.py

```diff
@@ -0,0 +1,10 @@
+# A tool to profile the ops in ONNX models
+# Run by providing path to an ONNX model as the only CLI arg
+# A set of all unique ops present in the model will be printed
+
+import onnx
+import sys
+
```

Comment:
Probably have main function and other functions?

We can do more profile on the model, not only op_types, we can keep it extensible